client/src/api/common/data.ts

```diff
@@ -76,8 +76,7 @@ export function useSensorData<T extends SensorsT>(
   const module = useModuleData(id);
 
   // Data from sensor
-  const data =
-    module.sensors.find((s) => s.type === sensor)?.value ?? null; // Data for the sensor // Extract value, defaulting to null
+  const data = module.sensors.find((s) => s.type === sensor)?.value ?? null; // Data for the sensor // Extract value, defaulting to null
```

Comment:
I think prettier messed up the indenting here. Maybe combine the comments or use `// prettier-ignore`.